client/app/templates/show-geography.hbs

```diff
@@ -396,6 +396,66 @@
           </ul>
         {{/section.filter-wrapper}}
       {{/filters.section}}
+
```

Comment:
Add a feature flag environment variable to use here to hide this. I don't _think_ you'll need conditional logic anywhere else but do some testing with the feature hidden to make sure it doesn't cause any regressions.

Reply:
Done.